File: bounce/src/main/java/is/ricko/bounce/BounceApplication.java
package is.ricko.bounce;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackageClasses = BounceApplication.class)
public class BounceApplication {
  public static void main(final String[] args) {
    SpringApplication.run(BounceApplication.class, args);
  }
}

File: bounce/src/main/kotlin/is/ricko/bounce/BounceApplication.kt
package `is`.ricko.bounce

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [BounceApplication::class])
class BounceApplication

fun main(args: Array<String>) {
    runApplication<BounceApplication>(*args)
}
